[Docs] Add VBUS hack for teensys (#10986)
qmk/qmk_firmware: docs/feature_split_keyboard.md

```
This sets the poll frequency when detecting master/slave when using `SPLIT_USB_DETECT`

> ## Hardware Considerations and mods

Reviewer: ```suggestion
## Hardware Considerations and Mods
```

> While most any Pro Micro can be used, micro controllers like the AVR teensy's and most (if not all) ARM boards require the Split USB Detect.

Reviewer: ```suggestion
While most any Pro Micro can be used, micro controllers like the AVR Teensys and most (if not all) ARM boards require the Split USB Detect.
```

However, with the Teensy 2.0 and Teensy++ 2.0, there is a simple hardware mod that you can perform to add VBUS detection, so you don't need the Split USB detection option.

You'll only need a few things:

* A knife (x-acto knife, ideally)
* A solder station or hot air station

> * An appropriate Schokkty diode, such as the [PMEG2005EH](https://www.digikey.com/en/products/detail/nexperia-usa-inc/PMEG2005EH,115/1589924)

Reviewer 2: ```suggestion
* An appropriate Schottky diode, such as the [PMEG2005EH](https://www.digikey.com/en/products/detail/nexperia-usa-inc/PMEG2005EH,115/1589924)
```

You'll need to cut the small trace between the 5V and center pads on the back of the Teensy.

Once you have done that, you will want to solder the diode from the 5V pad to the center pad.

> I had to use the 5V pad from the regulator block above as the pads were too small and placed too closely together.

Reviewer: Not sure about the use of first person in docs :P

Author: Ah, yeah, true. 

```suggestion
You may need to use the 5V pad from the regulator block above as the pads were too small and placed too closely together to place the Schottky diode properly. 
```

Should I also mention that I had NO issue with the size on the fake teensy? Only my genuine teensys had this issue?

> ![Teensy++ 2.0](https://cdn.discordapp.com/attachments/440911374571274240/779382993289871360/20201120_082838.jpg)

Reviewer: ```suggestion
![Teensy++ 2.0](https://i.imgur.com/BPEC5n5.png)
```
